### PR TITLE
new field type CpsiMapview.view.field.YearField

### PR DESCRIPTION
**Pull request by leonardospina**
Custom field to display years in a YYYY format. Displays text aligned to right.

### CODE REVIEW

File: app/view/field/YearField.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Ext.define('CpsiMapview.view.field.YearField', {
+    extend: 'Ext.form.field.Number',
+    xtype: 'cmv_yearfield',
+    fieldStyle: 'text-align: right;', // TODO has no effect?
```
Comment (geographika) [inline app/view/field/YearField.js:4]: Can you check if this is still a valid ExtJS property? If not then it should be removed

File: app/view/field/YearField.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    // textAlign: 'right', // property not available
```
Comment (geographika) [inline app/view/field/YearField.js:5]: Remove commented code

**Review by geographika — CHANGES_REQUESTED** [review]

**Comment by geographika** [comment]
Thanks @leonardospina - could you add a brief description of the change as part of the pull request?

File: app/view/field/YearField.js (hunk 1 above)
Comment (leonardospina) [reply app/view/field/YearField.js]: It is!

File: app/view/field/YearField.js (hunk 2 above)
Comment (leonardospina) [reply app/view/field/YearField.js]: Commented code removed

**Review by geographika — APPROVED** [review]

**Pull request merged**